hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RocksDBConfiguration.java

```diff
@@ -46,6 +46,20 @@ public class RocksDBConfiguration {
       description = "OM RocksDB logging level (INFO/DEBUG/WARN/ERROR/FATAL)")
   private String rocksdbLogLevel;
 
+  @Config(key = "rocksdb.max.log.file.size",
+      type = ConfigType.SIZE,
+      defaultValue = "0MB",
```

Comment:
Does 0MB denote unlimited size? Is it the current default for rocksdb?

Reply:
Yes, [max_log_file_size](https://github.com/facebook/rocksdb/blob/main/include/rocksdb/options.h#L864) as mentioned in the description.